### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by debopamsengupta**
Hi @matteofigus ,
I added some integration tests, to ensure full Windows support.

Let me know if you feel anything else might be important :)

### CODE REVIEW

**Comment by matteofigus** [comment]
Looks good! Need to find out what's going on with Windows. My assumption is that the require stuff fails due to some path normalisation that is not right for Windows.

Another thing to test, not sure it is supported out of the box, is the possibility to require different file trees, such as `require('lodash/package.json')` or `require('lodash@version/package.json')` which is something we worked on here https://github.com/opentable/oc/pull/323

File: integration/integration.test.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    expect(a).toBeTruthy();
+    expect(b).toBeTruthy();
+    expect(c).toBeTruthy();
+  });
+
+  it('should fetch correct version', () => {
+    let requirer = new ry.Requirer();
+    let a = requirer.require('lodash@1.3.1');
+    expect(a.drop).toBeTruthy(); //for 1.x
```
Comment (debopamsengupta) [inline integration/integration.test.js]: just noticed `lodash` has a VERSION property available 😂  will update the tests

**Comment by debopamsengupta** [comment]
@matteofigus Finally discovered the issue with windows , which is https://github.com/scott113341/npm-install-version/issues/32

Will wait on that to get merged, or would it be better to rewrite the installation/requiring logic in this project ?

**Comment by matteofigus** [comment]
Let's see if they reply and merge. If they take too much to reply, probably we can fork or re-implement (it should be not too much code for that).

**Review by matteofigus — APPROVED** [review]
🎉

**Pull request merged**